Google.PowerShell/Logging/GcLogSinkCmdlets.cs

```diff
@@ -317,6 +267,217 @@ protected override void ProcessRecord()
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> New-GcLogSink -SinkName "my-sink" -GcsBucketDestination "my-bucket"</code>
+    ///   <para>
+    ///   This command creates a sink name "my-sink" that exports every log entry in the default project to the
+    ///   Google Cloud Storage bucket "my-bucket". The identity of the writer of the logs will be cloud-logs@system.gserviceaccount.com.
+    ///   </para>
+    /// </example>
+    /// <example>
+    ///   <code>
+    ///   PS C:\> New-GcLogSink -SinkName "my-sink" -BigQueryDataSetDestination "my_dataset" -LogName "my-log" -Project "my-project"
+    ///   </code>
+    ///   <para>
+    ///   This command creates a sink name "my-sink" that exports every log entry in the log "my-log" in the
+    ///   project "my-project" to the Google Cloud BigQuery dataset "my_dataset" (also in the project "my-project").
+    ///   The identity of the writer of the logs will be cloud-logs@system.gserviceaccount.com.
+    ///   </para>
+    /// </example>
+    /// <example>
+    ///   <code>
+    ///   PS C:\> New-GcLogSink -SinkName "my-sink" -PubSubTopicDestination "my_dataset" -ResourceType gce_instance -After [DateTime]::Now().AddDays(1)
+    ///   </code>
+    ///   <para>
+    ///   This command creates a sink name "my-sink" that exports every log entry of the resource type gce_instance that is created the next day
+    ///   onwards to the Google Cloud PubSub topic "my-topic". The identity of the writer of the logs will be cloud-logs@system.gserviceaccount.com.
+    ///   </para>
+    /// </example>
+    /// <example>
+    ///   <code>
+    ///   PS C:\> New-GcLogSink -SinkName "my-sink" -PubSubTopicDestination "my_dataset" -Filter 'textPayload = "textPayload"' -UniqueWriterIdentity
+    ///   </code>
+    ///   <para>
+    ///   This command creates a sink name "my-sink" that exports every log entry that matches the provided filter to
+    ///   the Google Cloud PubSub topic "my-topic". The identity of the writer of the logs will be a unique service account.
+    ///   </para>
+    /// </example>
+    /// <para type="link" uri="(https://cloud.google.com/logging/docs/basic-concepts#sinks)">
+    /// [Log Sinks]
+    /// </para>
+    /// <para type="link" uri="(https://cloud.google.com/logging/docs/export/using_exported_logs)">
+    /// [Exporting Logs]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "GcLogSink")]
+    public class NewGcLogSinkCmdlet : CreateOrSetGcLogSinkCmdlet
+    {
+        /// <summary>
+        /// <para type="description">
+        /// Determines the kind of IAM identity returned as writerIdentity in the new sink.
+        /// If this value is provided, then the value returned as writerIdentity is cloud-logs@google.com.
+        /// Otherwise, it will be a unique service account.
+        /// </para>
+        /// </summary>
+        [Parameter(Mandatory = false)]
+        public override SwitchParameter UniqueWriterIdentity { get; set; }
```

Comment:
Do you need to restate this parameter? I don't see anything that changed from the parent (including the comment).
Alternatively, perhaps make the overridden parameter abstract, and keep the differing comments at the top level classes?